tabcmd/commands/user/user_data.py

```diff
@@ -75,7 +75,13 @@ def to_tsc_user(self) -> TSC.UserItem:
     "Unlicensed",
 ]
 
-auth_types = ["Local", TSC.UserItem.Auth.SAML, TSC.UserItem.Auth.OpenID, TSC.UserItem.Auth.ServerDefault, "TableauId"]
+auth_types = [
+    "Local",
+    TSC.UserItem.Auth.SAML,
+    TSC.UserItem.Auth.OpenID,
+    TSC.UserItem.Auth.ServerDefault,
+    TSC.UserItem.Auth.TableauIDWithMFA,
```

Comment:
Removal of "TableauId" intentional?

Reply:
They can  - I'm not sure how that works, but these are the only four Auth types in the server now.